src/components/Todo/TodoCards/todo-action-bar.component.jsx

```diff
@@ -5,12 +5,21 @@ import editButton from "assets/images/edit.png";
 import deleteButton from "assets/images/delete.png";
 import { COMPLETE_BUTTON, EDIT_BUTTON, DELETE_BUTTON } from "common/constants";
 
-const TodoActionBar = ({ handleDeleteTask, todo }) => {
-  const { id } = todo;
+const TodoActionBar = ({ handleDeleteTask, handleCompleteTask, todo }) => {
```

Comment:
see https://github.com/iud-wtag/Todos/pull/3/files#r1172120015

```suggestion
const TodoActionBar = ({ onDeleteTask, onCompleteTask, todo }) => {
```